python_modules/libraries/dagster-airlift/dagster_airlift/in_airflow/base_asset_operator.py

```diff
@@ -6,8 +6,15 @@
 from typing import Any, Dict, Iterable, Mapping, Sequence, Tuple
 
 import requests
-from airflow.models.operator import BaseOperator
-from airflow.utils.context import Context
+from airflow.models import BaseOperator
+
+try:
+    # Attempt to import the Context type from Airflow 2
+    from airflow.utils.context import Context
+except ImportError:
+    # Fallback for Airflow 1: Use a generic dictionary type as a substitute
+    Context = Dict[str, Any]  # Define Context as a dictionary
```

Comment:
Rather than do the try-catch; I think it's simpler to just declare as Dict[str, Any] and leave a comment about 1.0 compatibility.